frontend/src/model/beta/overview/ModelCardRevisionListDisplay.tsx

```diff
@@ -0,0 +1,31 @@
+import { TableBody, TableCell, TableRow } from '@mui/material'
+import { ThemeProvider, useTheme } from '@mui/material/styles'
+import { useRouter } from 'next/router'
+
+import { ModelCardRevisionInterface } from '../../../../types/v2/types'
+import { formatDateString } from '../../../../utils/dateUtils'
+
+type revisionProp = {
+  modelCard: ModelCardRevisionInterface
+}
+export default function ModelCardRevisionListDisplay({ modelCard }: revisionProp) {
+  const router = useRouter()
+  const { modelId, modelCardVersion }: { modelId?: string; modelCardVersion?: number } = router.query
+  const theme = useTheme()
+
+  return (
+    <ThemeProvider theme={theme}>
+      <TableBody>
+        <TableRow
+          onClick={() => router.push(`/beta/model/${modelId}/history/${modelCardVersion}`)}
+          // this route leads to blank page that says 'undefined'
+          sx={{ '&:hover': { backgroundColor: '#f0f0f0', cursor: 'pointer' } }}
```

Comment:
We shouldn't really use custom codes like this and should instead change our theme. Happy to help with this tomorrow if you need

Reply:
Actually, you can remove `backgroundColor: : '#f0f0f0'` entirely, and add a prop called `hover` to your `<TableRow />`